internal/controller/suite_test.go

```diff
@@ -112,7 +112,7 @@ var _ = BeforeSuite(func() {
 		Client:          mgr.GetClient(),
 		Scheme:          mgr.GetScheme(),
 		ProviderFactory: providerFactory,
-	}).SetupWithManager(mgr, RequeueDuration, ValidityDuration)
+	}).SetupWithManager(mgr, RequeueDuration, ValidityDuration, DefaultValidationDuration)
```

Comment:
What is the relationship between these? Can the controller be started with any combination of values of these things and behave in a an appropriate way?